etc/picongpu/pizdaint-cscs/picongpu.profile.example

```diff
@@ -25,19 +25,29 @@ else
     module load PrgEnv-gnu/6.0.4
 fi
 
-module load cudatoolkit/8.0.61_2.4.3-6.0.4.0_3.1__gb475d12
+module load daint-gpu
```

Comment:
after the above case:
```
$ module load daint-gpu
daint-gpu(31):ERROR:105: Unable to locate a modulefile for 'craype-haswell'
```

Reply:
See above, `module purge` should *not* be done on cray systems.